CHANGELOG.md

```diff
@@ -8,9 +8,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Updated EncyclopeDIA parameters to match GUI defaults.
 - Retries should automatically occur when MSconvert hangs
+- The library file used for the EncyclopeDIA global step is now the DLIB/ELIB
+  the files were searched against. This has not affect on workflows that were
+  not using a chromatogram library.
```

Comment:
I am having a hard time understanding what this is saying. This is what I think you are saying:
"The library file used for the EncyclopeDIA global step is now the DLIB/ELIB
 the files were searched against in the local step. This only affects workflows that use a chromatogram library."